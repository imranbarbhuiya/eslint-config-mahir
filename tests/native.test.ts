import eslintConfig from '../native';

describe('ESLint Config', () => {
	test('should export rules', () => {
		expect(eslintConfig).toMatchSnapshot();
	});
});
