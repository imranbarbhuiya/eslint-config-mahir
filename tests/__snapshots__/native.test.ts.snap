// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ESLint Config > should export rules 1`] = `
{
  "extends": "./react.js",
  "globals": {
    "Atomics": false,
    "ErrorUtils": false,
    "FormData": false,
    "SharedArrayBuffer": false,
    "XMLHttpRequest": false,
    "__DEV__": false,
    "alert": false,
    "cancelAnimationFrame": false,
    "cancelIdleCallback": false,
    "clearImmediate": false,
    "clearInterval": false,
    "clearTimeout": false,
    "console": false,
    "exports": false,
    "fetch": false,
    "global": false,
    "module": false,
    "navigator": false,
    "process": false,
    "requestAnimationFrame": false,
    "requestIdleCallback": false,
    "require": false,
    "setImmediate": false,
    "setInterval": false,
    "setTimeout": false,
    "window": false,
  },
  "overrides": [
    {
      "env": {
        "browser": true,
      },
      "files": [
        "*.web.*",
      ],
    },
  ],
  "settings": {
    "import/ignore": [
      "node_modules[\\\\\\\\/]+@?react-native",
    ],
  },
}
`;
